vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -1173,6 +1189,14 @@
             }
         }
 
+        # Do a second-run of the incremental analysis to compare the result files
+        if ($check_incremental_sta_consistency) {
+            if (&cmp_full_vs_incr_STA ne "success") {
```

Comment:
I'd re-structure this as something like:

```
my $result = &cmp_full_vs_incr_STA();

if ($result ne "success") {
    $error_status = $result'
    $error_code = 1;
```

That both makes it clear that we are doing a function call, and preserves the value returned (which should provide details about the error as noted below)